laterpay/application/Controller/Admin/Settings.php

```diff
@@ -981,4 +1011,247 @@ public function get_laterpay_pro_merchant_description() {
         esc_html_e( 'Please choose, if you have a LaterPay Pro merchant account.', 'laterpay' );
         echo '</p>';
     }
+
+    /**
+     * Method to render ga fields.
+     *
+     * @param array $fields array of field params.
+     *
+     * @return void
+     */
+    public function get_ga_field_markup( $fields = null ) {
```

Comment:
@thrijith Should this be part of view?

Reply:
This is modelled after https://github.com/laterpay/laterpay-wordpress-plugin/blob/86011b7bcbfbde68f5c46a8a38e1c99ea0f5095d/laterpay/application/Controller/Admin/Settings.php#L581-L582

and is a callback for `add_settings_field `, GA fields needed customizations so created a new one for it. It generates markup based on field/values passed.

Let me know if this needs to be changed.

Reply:
Is there any model for backend UI? If then we should try to move if possible.

Reply:
Nope, not right now. Everything on the setting page is added via https://github.com/laterpay/laterpay-wordpress-plugin/blob/86011b7bcbfbde68f5c46a8a38e1c99ea0f5095d/laterpay/application/Controller/Admin/Settings.php#L86